dijet/production/jet_assignment.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+
+"""
+Selectors to set ak columns for dijet properties
+"""
+
+# TODO: Not all columns needed are present yet for pt balance method
+#       - Include probe and reference jet from jet assignment
+#       - is FE or SM Method
+
+from columnflow.util import maybe_import
+from columnflow.columnar_util import set_ak_column
+from columnflow.production import Producer, producer
+from columnflow.calibration.util import ak_random
+
+from dijet.constants import eta
+
+np = maybe_import("numpy")
+ak = maybe_import("awkward")
+
+"""
+Creates column probe and reference jets, used in dijet analysis.
+Only store pt, eta and phi field for now and remove mass and pdgId
+"""
+
+
+@producer(
+    uses={
+        "Jet.pt", "Jet.eta",
+    },
+    produces={
+        "use_sm", "use_fe", "n_jet",
+        "probe_jet.pt", "probe_jet.eta", "probe_jet.phi",
+        "reference_jet.pt", "reference_jet.eta", "reference_jet.phi",
```

Comment:
It might be wise to also store the `mass` so that you can reconstruct the full four-vector in a later analysis stage, should the need ever arise:
```suggestion
        "probe_jet.pt", "probe_jet.eta", "probe_jet.phi", "probe_jet.mass",
        "reference_jet.pt", "reference_jet.eta", "reference_jet.phi", "reference_jet.mass",
```